### ISSUE TITLE
Strange Name in the Navigation Property

### ISSUE THREAD

**Issue by caiuaqui** [ErikEJ/EFCorePowerTools#1409]
Good morning.

I came across another weird issue, this time on the name of the properties refering to the foreign keys.

**DB**
```
CREATE TABLE [dbo].[Menu](
	[Organizacao] [varchar](15) NOT NULL,
	[Opcao] [varchar](30) NOT NULL,
	[Menu] [varchar](30) NULL,
	......
 CONSTRAINT [PK_Menu] PRIMARY KEY CLUSTERED 
(
	[Organizacao] ASC,
	[Opcao] ASC
)

CREATE TABLE [dbo].[PerfilMenu](
	[Organizacao] [varchar](15) NOT NULL,
	[Perfil] [varchar](30) NOT NULL,
	[Opcao] [varchar](30) NOT NULL,
	.........
CONSTRAINT [PK_PerfilMenu] PRIMARY KEY CLUSTERED 
(
	[Organizacao] ASC,
	[Perfil] ASC,
	[Opcao] ASC
)

ALTER TABLE [dbo].[PerfilMenu]  WITH CHECK ADD  CONSTRAINT [FK_PerfilMenu_Menu] FOREIGN KEY([Organizacao], [Opcao])
REFERENCES [dbo].[Menu] ([Organizacao], [Opcao])
GO
ALTER TABLE [dbo].[PerfilMenu] CHECK CONSTRAINT [FK_PerfilMenu_Menu]

ALTER TABLE [dbo].[PerfilMenu]  WITH CHECK ADD  CONSTRAINT [FK_PerfilMenu_Perfil] FOREIGN KEY([Organizacao], [Perfil])
REFERENCES [dbo].[Perfil] ([Organizacao], [Perfil])
GO
ALTER TABLE [dbo].[PerfilMenu] CHECK CONSTRAINT [FK_PerfilMenu_Perfil]
```
**Generated Code**
```
[Table("PerfilMenu")]
public partial class PerfilMenus
{
	[Key]
	[StringLength(15)]
	public string Organizacao { get; set; }
	[Key]
	[StringLength(30)]
	public string Perfil { get; set; }
	[Key]
	[StringLength(30)]
	public string Opcao { get; set; }
        ...........................

	[ForeignKey("Organizacao,Opcao")]
	[InverseProperty(nameof(Menus.PerfilMenus))]
	public virtual Menus O { get; set; }
	[ForeignKey("Organizacao,Perfil")]
	[InverseProperty("PerfilMenus")]
	public virtual Perfis Perfis { get; set; }
}
```
Any reason as to why the property `Menus` in the PerfilMenus class is named `O` as opposing the property Perfis in which the name is correct?

These are the options used to generate the code.
![image](https://user-images.githubusercontent.com/6917622/170738979-c0401d03-2f4f-441a-99ec-aa426e79e1bf.png)
![image](https://user-images.githubusercontent.com/6917622/170739333-9782caf2-a03e-4121-8539-8070ca4832e3.png)

Technical Details:
EF Core version in use: EF Core 5
Is Handlebars used: no
EF Core Power Tools version: 2.5.982
Database engine: SQL Server
Visual Studio version: Visual Studio 2019

**Comment by ErikEJ** [comment]
Seems like a duplicate of #1342 - which is an EF Core bug

**Comment by caiuaqui** [comment]
Well, it most likely is.
And unfornately this issue will not be addressed until the 7.0.0 version of EF Core.
I'm closing this issue.
